rel/overlay/etc/default.ini

```diff
@@ -145,6 +145,7 @@ max_db_number_for_dbs_info_req = 100
 ; can be the name of a claim like "exp" or a tuple if the claim requires
 ; a parameter
 ; required_claims = exp, {iss, "IssuerNameHere"}
+; roles_claim_name = my_custom_claim_containing_roles_array
```

Comment:
in what way is it an array?

Reply:
It's the value of the claim the one that actually contains the roles arra, just like in https://docs.couchdb.org/en/stable/api/server/authn.html#jwt-authentication
Do you have any better suggestion? Thanks.

Reply:
the value of the token is an array, yes, but roles_claim_name is a string.

suggest
`roles_claim_name = https://example.com/roles`

to match your PR comment.